cli/src/stake.rs

```diff
@@ -571,8 +571,8 @@ impl StakeSubCommands for App<'_, '_> {
                         .index(2)
                         .value_name("SOURCE_STAKE_ACCOUNT_ADDRESS")
                         .required(true),
-                    "Source stake account for the merge.  If successful, this stake account will \
-                     no longer exist after the merge"
+                    "Source stake account for the merge (if successful, this stake account will \
+                     no longer exist after the merge)"
```

Comment:
I see your point about feeling like info is being "demoted" here. I think things are fine as-is, but here are two more ideas for the brainstorm:
1. Adjust the macro to take an optional argument to put the text string at the end. So, final text looks like:
>     <ADDRESS>...    Source stake account for the merge, one of:
>                      * a base58-encoded public key
>                      * a path to a keypair file
>                      ...
>                     If successful, this stake account will no longer exist after the merge.

2. Adjust the macro definition to assume previous sentence had ended and add in the following:
> Address is one of:

Option 2. would make the complete text look like (CLAP will wrap the first line automatically I believe, I wrapped it for ease of reading this)
>     <ADDRESS>...    Source stake account for the merge. If successful, this stake account will 
>                     no longer exist after the merge. Address is one of:
>                      * a base58-encoded public key
>                      * a path to a keypair file
>                      ...

Reply:
Great thoughts! I went back and forth between your suggestions, but ultimately I prefer option 2 for ease of use at the call site, as well as shorter sentences overall. Will write it up. Because that change will make the first commit in this PR completely obsolete, I'm going to go ahead and force-push it to oblivion.

Reply:
Done. Let me know what you think!

Reply:
I like it!